owncloudApp/src/main/java/com/owncloud/android/presentation/accounts/AccountsManagementActivity.kt

```diff
@@ -2,8 +2,9 @@
  * ownCloud Android client application
  *
  * @author Javier Rodríguez Pérez
+ * @author Aitor Ballesteros Pavón
  *
- * Copyright (C) 2022 ownCloud GmbH.
+ * Copyright (C) 2024 ownCloud GmbH.
```

Comment:
When we're just removing content, there's no need to update the header, you didn't add any new content and so you are not author of anything in this class

Reply:
Done